token/cli/src/command.rs

```diff
@@ -3484,6 +3485,45 @@ pub async fn process_command<'a>(
                 )
             });
 
+            let transfer_fee_basis_point = arg_matches.values_of("transfer_fee_basis_points").map(|mut v| {
+                v.next()
+                    .unwrap()
+                    .parse::<u16>()
+                    .unwrap_or_else(print_error_and_exit)
+            });
+
+            let transfer_fee_maximum_fee = arg_matches.values_of("transfer_fee_maximum_fee").map(|mut v| {
+                v.next()
+                    .unwrap()
+                    .parse::<f64>()
+                    .unwrap_or_else(print_error_and_exit)
+            });
+
+            let transfer_fee = match transfer_fee.is_some() {
+                true => match transfer_fee_basis_point.is_some() || transfer_fee_maximum_fee.is_some() {
+                    true => Err("transfer-fee-basis-point and transfer-fee-maximum-fee flag disable when using transfer-fee flag"),
+                    false => Ok(transfer_fee)
+                },
+                false => match transfer_fee_basis_point.is_some() && transfer_fee_maximum_fee.is_some() {
+                    true => Ok(Some((transfer_fee_basis_point.unwrap(), (transfer_fee_maximum_fee.unwrap() * (u64::pow(10, decimals.into())) as f64) as u64))),
+                    false =>  if transfer_fee_basis_point.is_some() {
+                        Err("Missing transfer_fee_maximum_fee flag")
+                    }else if transfer_fee_maximum_fee.is_some() {
+                        Err("Missing transfer_fee_basis_point")
+                    }else{
+                        Ok(None)
+                    }
+                }
+            };
+
+            let transfer_fee = match transfer_fee {
+                Ok(result) => result,
+                Err(e) => {
+                    eprintln!("{}",e);
+                    exit(1);
+                }
+            };
```

Comment:
Here we'll need to combine the transfer fee to make it ready for the call to `command_create_token`, ie

```suggestion
            let transfer_fee = transfer_fee_basis_points.map(|v| (v, transfer_fee_maximum_fee.unwrap()).or(transfer_fee);
```

Maybe you can come up with something better!